### DISCUSSION TITLE
Grid implementation mismatch between @carbon/grid & carbon-components packages

### DISCUSSION THREAD

**Discussion by clhenrick** [carbon-design-system/carbon#10471] [Help] [answered]
I recently noticed that the bx grid implementation for the `@carbon/grid` package doesn't seem to follow the same implementation for the `carbon-components` package. The `@carbon/grid` package still points to the documentation that instructs using the `bx--grid`, `bx--row`, and `bx--col` CSS classes. However these classes do not seem to work out of the box with the `@carbon/grid` package. It would be helpful to update the docs for `@carbon/grid` for how to use it correctly.

**Comment by tay1orjones** [comment, marked as answer]
@clhenrick Sorry about this, I think you caught the package in a state where we had updated pieces for v11 to include the new CSS Grid implementation, and other pieces were not updated. It should be straightened out now. 

`@carbon/grid` includes both the existing (v10) flexbox-based grid and the new CSS Grid implementation scoped under the `.cds--css-grid` class.
